widgets/nuxeo-selectivity.html

```diff
@@ -513,19 +519,25 @@
       },
 
       _wrap: function(value) {
-        var fn = function(item) {
+        var fn = function(item, level) {
+          var depth = level || 0;
           var id = this.idFunction(item);
           var entry = {
             id: id,
             text: item.displayLabel || item.text || id || '',
-            item: item
+            item: item,
+            depth: depth
           };
           if (item.children) {
-            entry.children = item.children.map(fn.bind(this));
+            entry.children = item.children.map(function(child) { return fn(child, entry.depth + 1) });
```

Comment:
nit: missing semicolon at the end of return statement:
```suggestion
            entry.children = item.children.map(function(child) { return fn(child, entry.depth + 1); });
```